tests/jest.config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "testEnvironment": "node"
+}
```

Comment:
Please add a newline to the end of the file. Otherwise, PR looks great :)

Reply:
@AmyJZhao , fixed